package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@jupiterone/integration-sdk",
+  "version": "0.0.1",
```

Comment:
Another minor thing, following semver this should be `0.1.0` (I think, pls correct if wrong).

Reply:
I think you are right, since there's nothing to release yet, mind if the change to `0.1.0` happens when there's actually an initial publish?

Reply:
Sounds gucc